packages/expo-asset/package.json

```diff
@@ -37,7 +37,7 @@
     "preset": "expo-module-scripts"
   },
   "dependencies": {
-    "@react-native/assets-registry": "~0.74.83",
+    "@react-native/assets-registry": "~0.74.84",
```

Comment:
sorry i landed this workaround beforehand https://github.com/expo/expo/pull/29441 please help to resolve the conflicts